### PR TITLE
replaces classes for list, anchor and paragraph elements excluding instruction-list

### PR DESCRIPTION
**Pull request by agalamatis**
trello: https://trello.com/c/pNq68dXy/78-2-use-govuk-link-govuk-list-and-govuk-body-styles-in-briefs-frontend

### CODE REVIEW

File: app/templates/buyers/dashboard.html [hunk 1]
```diff
@@ -111,9 +111,9 @@
           {{ summary.text(item.applicationsClosedAt|dateformat) }}
           {% call summary.field(action=True) %}
             <div class="padding-bottom-small">
-                <p><a href="{{ url_for('.view_brief_responses', framework_slug=item.frameworkSlug, lot_slug=item.lot, brief_id=item.id) }}">View responses</a></p>
+                <p class="govuk-body"><a class="govuk-link" href="{{ url_for('.view_brief_responses', framework_slug=item.frameworkSlug, lot_slug=item.lot, brief_id=item.id) }}">View responses</a></p>
                 {% if item.status == "closed" %}
-                    <p><a class="award-contract-link" href="{{ url_for('.award_or_cancel_brief', framework_slug=item.frameworkSlug, lot_slug=item.lot, brief_id=item.id) }}">Let suppliers know the outcome</a></p>
+                    <p class="govuk-body"><a class="govuk-link" class="award-contract-link" href="{{ url_for('.award_or_cancel_brief', framework_slug=item.frameworkSlug, lot_slug=item.lot, brief_id=item.id) }}">Let suppliers know the outcome</a></p>
```
Comment (lfdebrux) [inline app/templates/buyers/dashboard.html]: This link already has a `class` attribute; can we get rid of it?

```suggestion
                    <p class="govuk-body"><a class="govuk-link" href="{{ url_for('.award_or_cancel_brief', framework_slug=item.frameworkSlug, lot_slug=item.lot, brief_id=item.id) }}">Let suppliers know the outcome</a></p>
```

File: app/templates/create_buyer/create_buyer_user_error.html [hunk 2]
```diff
@@ -53,30 +53,30 @@ <h1>Expired link</h1>
     <header class="page-heading-smaller">
       <h1>Your account has been deactivated.</h1>
     </header>
-    <p>
-      Email <a href="mailto:{{ support_email_address }}">{{ support_email_address }}</a> to reactivate your account.
+    <p class="govuk-body">
+      Email <a class="govuk-link" class="govuk-link" href="mailto:{{ support_email_address }}">{{ support_email_address }}</a> to reactivate your account.
```
Comment (lfdebrux) [inline app/templates/create_buyer/create_buyer_user_error.html]:
```suggestion
      Email <a class="govuk-link" href="mailto:{{ support_email_address }}">{{ support_email_address }}</a> to reactivate your account.
```

File: app/templates/buyers/dashboard.html (hunk 1 above)
Comment (risicle) [reply app/templates/buyers/dashboard.html]: My guess is it's there for a reason...

File: app/templates/create_buyer/create_buyer_account.html [hunk 3]
```diff
@@ -32,7 +32,7 @@
 {% include "toolkit/page-heading.html" %}
 {% endwith %}
 
-<p class="lede">
+<p class="govuk-body-l">
   You must be employed by, or represent, a public sector organisation.
 </p>
```
Comment (lfdebrux) [inline app/templates/create_buyer/create_buyer_account.html]: Missed on at L45 of this file https://github.com/alphagov/digitalmarketplace-briefs-frontend/blob/bcba80b930d91fbf5353b50b59e3b71387d2a88e/app/templates/create_buyer/create_buyer_account.html#L45

**Pull request merged**